test/runtests.jl

```diff
@@ -1,6 +1,30 @@
+import Pkg
+
+using Plots
+using VisualRegressionTests
 using SpikingNN
 using Test
 
+# environment settings
+isci = "CI" ∈ keys(ENV)
+datadir = joinpath(@__DIR__, "data")
+if !isci
+    Pkg.add("Gtk")
+    using Gtk
+end
+
+# test files to include
+testfiles = [
+    "excite-test.jl",
+    "lif-test.jl",
+    "population-test.jl",
+    "srm0-test.jl",
+    "srm0-threshold-test.jl",
+    "stdp-test.jl"
+]
+
 @testset "SpikingNN.jl" begin
-    # Write your tests here.
+    for testfile in testfiles
```

Comment:
Can you add the environment variable for GR as described [here](https://github.com/jheinen/GR.jl/issues/278#issuecomment-563129986)